library/core/game_test/src/main/java/org/quiltmc/qsl/game_test/api/package-info.java

```diff
@@ -0,0 +1,117 @@
+/**
+ * <h2>The Game Test API.</h2>
+ *
+ * <p>
+ * <h3>What is the Game Test API?</h3>
+ * The Game Test API is the test automation framework used by Minecraft as most test frameworks are not adapted to a game that's so dynamic.
+ * Unit testing presents very quickly its issues: you will quickly find out that testing while in a tick loop is very difficult.
+ * Thus Mojang made their own testing framework, watching the <a href="https://www.youtube.com/watch?v=TNkPE6NTNHQ">keynote by Henrik Kniberg</a>
+ * is highly recommended to grasp the why, and how it works.
```

Comment:
```suggestion
 * The Game Test API is the custom automated test framework used by Minecraft.
 * Minecraft uses a custom framework because most pre-existing ones do not adapt well to such a dynamic environment as Minecraft. 
 * Thus Mojang made their own testing framework. Watching the <a href="https://www.youtube.com/watch?v=TNkPE6NTNHQ">keynote by Henrik Kniberg</a>
 * is highly recommended to grasp how it works and why it was designed that way.
```